### PR TITLE
Refresh /ceph/install

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Refreshes /ceph/install based on [copydoc](https://docs.google.com/document/d/1EmOtHjuEMKhVLQA7GwyrKfCzOKfc8UCEyh-p04-PxzA/edit)

## QA

- Check that [the demo](https://ubuntu-com-12976.demos.haus/ceph/install) content matches the [copydoc](https://docs.google.com/document/d/1EmOtHjuEMKhVLQA7GwyrKfCzOKfc8UCEyh-p04-PxzA/edit)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4453

### CODE REVIEW

**Review by mtruj013 — APPROVED** [review]
LGTM, with some small changes. There are some copy changes in the hero section that need addressing and the "Single-node, multi-node, containerised or large-scale Cluster?" heading is missing

**Comment by camillet16** [comment]
LGTM

File: templates/ceph/install.html [hunk 1]
```diff
@@ -33,112 +33,248 @@ <h1>Install Ceph on Ubuntu</h1>
...
+              </h3>
+              <div class="p-stepped-list__content">
+                <div class="p-code-snippet">
+                  <pre class="p-code-snippet__block"><code>sudo microceph.ceph status</code></pre>
+                </div>
+              </div>
+            </li>
+            <li class="p-stepped-list__item">
+              <h3 class="p-stepped-list__title p-heading--4">
+                Here you should see that there is a single node in the cluster. <br/>
```
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/ceph/install.html:102]: Do you mean this sentence should be in the list item above it?

**Comment by juanruitina** [comment]
Some comments:

- Can you please make the lists of features below the tabs look like the ones [here](https://ubuntu.com/openstack/install)? Checkboxes, no border.
- Styling of the steps is fine, but I think we don't need to use header tags for these, a normal `p` would be enough: it is a list already, so markup is good enough. Would you agree?
- Step 4 under Single-node and Multi-node, please make the "Repeat for each disk..." line a regular paragraph, also in styling.

Thank you!

**Comment by anthonydillon** [comment]
@juanruitina can you please change the label to -1 if there are changes requested? This indicates to the author of the PR there is something to work on.

**Comment by juanruitina** [comment]
LGTM!

**Comment by petesfrench** [comment]
Waiting on a question to be answered in the doc

**Pull request merged**